xcube/webapi/places/context.py

```diff
@@ -183,27 +191,49 @@ def _load_place_group(self,
                                sourceEncoding=source_encoding,
                                join=join)
 
-            sub_place_group_configs = place_group_config.get("Places")
-            if sub_place_group_configs:
-                raise ApiError.InvalidServerConfig(
-                    "Invalid 'Places' entry in a 'PlaceGroups' item:"
-                    " not implemented yet"
-                )
-
-            # sub_place_group_configs = place_group_config.get("Places")
-            # if sub_place_group_configs:
-            #     sub_place_groups = self._load_place_groups(
-            #         sub_place_group_configs
-            #     )
-            #     place_group["placeGroups"] = sub_place_groups
-
+            self.check_sub_group_configs(place_group_config)
             self.set_cached_place_group(place_group_id, place_group)
 
         if load_features:
             self.load_place_group_features(place_group)
 
         return place_group
 
+    @staticmethod
+    def get_place_group_id_safe(place_group_config):
+        place_group_id = place_group_config.get("Identifier")
+        if not place_group_id:
+            raise ApiError.InvalidServerConfig(
+                "Missing 'Identifier' entry in a 'PlaceGroups' item"
+            )
+        return place_group_id
+
+    @staticmethod
+    def check_sub_group_configs(place_group_config):
+        sub_place_group_configs = place_group_config.get("Places")
+        if sub_place_group_configs:
+            raise ApiError.InvalidServerConfig(
+                "Invalid 'Places' entry in a 'PlaceGroups' item:"
+                " not implemented yet"
+            )
+        # sub_place_group_configs = place_group_config.get("Places")
```

Comment:
can this be removed? I understand, it was there before, but still I dont understand

Reply:
done